packages/strapi-plugin-graphql/services/GraphQL.js

```diff
@@ -323,12 +349,9 @@ module.exports = {
       connection: `${globalId}Connection`,
     }));
 
-    let connectionFieldsTypes = '';
-    _.forEach(primitiveFields, (fieldValue, fieldKey) => {
-      connectionFieldsTypes += `type ${globalId}Connection${_.upperFirst(fieldKey)} {${this.formatGQL(connectionFields[fieldKey])}}\n\n`;
-    });
-
-    return connectionFieldsTypes;
+    return Object.keys(primitiveFields).map((fieldKey) =>
+      `type ${globalId}Connection${_.upperFirst(fieldKey)} {${this.formatGQL(connectionFields[fieldKey])}}`
```

Comment:
You can remove the lodash function `upperFirst` too 😉

Reply:
? what do you want me to use instead?

Reply:
I confirm, `.chartAt()` was use for compatibility with old versions of IE, you can so use string[0] to get the first letter directly 👍